app/src/pages/inside/common/suiteTestToolbar/actionPanel/actionPanel.scss

```diff
@@ -22,3 +22,15 @@
     display: none;
   }
 }
+
+@media (min-width: $SCREEN_SM_MIN) and (max-width: 1195px) {
```

Comment:
Please, follow comment form the [link](https://github.com/reportportal/service-ui/pull/1071#discussion_r248955146)